apps/prairielearn/src/server.js

```diff
@@ -2025,6 +2025,21 @@ export async function initExpress() {
       coreElements: false,
     }),
   );
+  app.use(
+    '/pl/public/course_instance/:course_instance_id(\\d+)/assessment/:assessment_id(\\d+)/questions',
```

Comment:
Great question. For now I manually construct it. I have a merge request I'll add later that has a link to a course's `public course overview` page that shows all of that course's publicly shared assessments(will link to this page), questions, and course instances(pull request not opened yet). 

We can add a temporary link somewhere or add a simplified version of the `public course overview` page

Reply:
I think it'd be great to add such a link to the `instructorAssessmentSettings` page. It'd also be nice to display if it _is_ shared on that page, just like we do on the question settings page:

<img width="495" alt="Screenshot 2024-11-20 at 09 57 21" src="https://github.com/user-attachments/assets/99878228-8170-4dce-9448-cc95d915adc9">

This will be increasingly important as we shift towards GUI editors and away from editing raw JSON.

Reply:
oh ya, that's a great idea, I hadn't thought of that

Reply:
We should add the public question link on that for the publicly shared questions too, that would be awesome (not this PR though)

Reply:
Added link to `instructorAssessmentSettings` that shows up with the `Public source` badge *if* the assessment's `share_source_publicly` is true. Else we display it's not shared publicly